spec/acceptance/chain_key_spec.rb

```diff
@@ -182,30 +147,4 @@
       it { is_expected.not_to be_file }
     end
   end
-
-  describe 'managing existing java chain keys in noop', unless: UNSUPPORTED_PLATFORMS.include?(fact('operatingsystem')) do
```

Comment:
have these tests moved up or are not needed ?

Reply:
I folded them up into the first one in order to cut down on the number of manifest applies

Reply:
Or at least the first one, did not move this one. No idea why it is showing up red, It's still in place in my local copy and when viewing the file it has not disappeared.

Reply:
so the number of manifests run, has dropped from 6 to 3. is that because you have grouped 2 tests in one ?

Reply:
From 5 to 3, but yeh. Should update the test description though. Look's like it doesn't really match. A failure just appeared on Ubuntu 18.04 on adhoc so will do it a the same time as I fix it.